### DISCUSSION TITLE
Auto-delete reports after x years

### DISCUSSION THREAD

**Discussion by waltergambino** [globaleaks/globaleaks-whistleblowing-software#3788] [General]
I would like to know if it is possible to provide for the automatic deletion of reports after x years from the closing date.

Thank you

**Comment by evilaliv3** [comment]
Hello, we are about to release a feature that will enable to configure data retention policies on each closing state.

We expect to release the feature in the next two weeks.

**Reply by waltergambino** [reply]
great!
